.env

```diff
@@ -0,0 +1,3 @@
+VITE_PORT = "8080"
+
+VITE_WEBSOCKET_URL = "ws://0.0.0.0:8179/ws/conference/"
```

Comment:
Also, please add a "FRONTEND_DEBUG" environment variable and set the application mode according to it's value. It would be hyperuseful during deployment.

Reply:
I don't think that it needs to be realised now
Please create a new issue for it